diag_manager/README.md

```diff
@@ -59,27 +59,27 @@ ESM4_piControl
 ```
 
 ### 2.2 File Section
-The files are listed under the diagFiles section as a dashed array. 
+The files are listed under the diagFiles section as a dashed array.
 
 Below are the **required** keys needed to define each file.
-- **file_name** is a string that defines the name of the file. Do not add ".nc" and "tileX" to the filename as this will handle by FMS. 
-- **freq** is an integer that defines the frequency that data will be written. The acceptable values are:
-  - =-1: output at the end of the run only 
-  - =0: output every timestep 
-  - \>0: output frequency
-- **freq_units** is a string that defines the units of the frequency from above. The acceptable values are seconds, minutes, hours, days, months, years. 
-- **time_units** is a string that defines units for time. The acceptable values are seconds, minutes, hours, days, months, years. 
+- **file_name** is a string that defines the name of the file. Do not add ".nc" and "tileX" to the filename as this will handle by FMS.
+- **freq** is a string that defines the frequency and the units that data will be written.
+  - The acceptable values for freq are:
+    - =-1: output at the end of the run only
+    - =0: output every timestep
+    - \>0: output frequency
+  - The acceptable values for units are seconds, minutes, hours, days, months, years.
+- **time_units** is a string that defines units for time. The acceptable values are seconds, minutes, hours, days, months, years.
```

Comment:
So do I write "24hours" or "24 hours"?  You should specify that the frequency and units should have a space between them.